### DISCUSSION TITLE
flutter conference consult

### DISCUSSION THREAD

**Discussion by nocpbxchap** [ant-media/Ant-Media-Server#6326] [General]
Hi, i am doing some testing with conference on flutter app, on my current development we are using depreciated conference solution and flutter app is using the multitrack conference seems not compatible, how can we still using conference depreciated so we dont have to do major changes?

example if we connect to conference using depreciated not working say no people on conference and if we use multitrack we can join conference from flutter app.

**Comment by lastpeony** [comment]
I highly recommend you to switch multitrack conference.
Legacy conference is abandoned
